SUMMARY.md

```diff
@@ -79,14 +79,17 @@
       * [Username/Password](nats-server/configuration/securing_nats/auth_intro/username_password.md)
       * [TLS Authentication](nats-server/configuration/securing_nats/auth_intro/tls_mutual_auth.md)
       * [NKeys](nats-server/configuration/securing_nats/auth_intro/nkey_auth.md)
-      * [Accounts](nats-server/configuration/securing_nats/auth_intro/accounts.md)
-      * [JWTs](nats-server/configuration/securing_nats/auth_intro/jwt_auth.md)
       * [Authentication Timeout](nats-server/configuration/securing_nats/auth_intro/auth_timeout.md)
     * [Authorization](nats-server/configuration/securing_nats/authorization.md)
+    * [Multi Tenancy using Accounts](nats-server/configuration/securing_nats/accounts.md)
```

Comment:
The Multi-Tenancy link is broken nats-server/configuration/securing_nats/accounts.md

Reply:
Hey Ginger, are you sure? I moved the entire file.